TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/FinalPath.java

```diff
@@ -0,0 +1,34 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
+
+@Autonomous (group = "drive")
+public class FinalPath extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
+
+        drive.setPoseEstimate(new Pose2d(50, 50, Math.toRadians(270)));
+        Trajectory cool = drive.trajectoryBuilder(new Pose2d(50, 50, Math.toRadians(270)))
```

Comment:
what does this trajectory do? Can we have a better variable name?

Reply:
i fw cool and swag ngl

Reply:
yessir

Reply:
the trajectory moves it straightforward to the opposite corner of the field